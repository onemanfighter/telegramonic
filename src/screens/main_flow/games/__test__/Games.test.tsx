import { render } from '@testing-library/react';
import Games from '../Games';

describe('Games screen', () => {
  it('should render correctly', () => {
    const { container } = render(<Games />);

    expect(container).toMatchSnapshot();
  });
});
