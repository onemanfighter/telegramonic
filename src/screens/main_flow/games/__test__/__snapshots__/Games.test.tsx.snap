// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Games screen should render correctly 1`] = `
<div>
  <div>
    <h1>
      Games
    </h1>
  </div>
</div>
`;
